File: exemplo.pas
program exemplo;
var
  a, b, c: boolean;
begin
  a := true;
  b := false;
  c := a AND b; // c recebe falso
  writeln(c);
  c := a OR b; // c recebe verdadeiro
  writeln(c);
  c := NOT a; // c recebe falso
  writeln(c);
  c := a XOR b; // c recebe verdadeiro
  writeln(c);
end.

File: exemplo.java
public class Exemplo {
  public static void main(String[] args) {
    boolean a = true;
    boolean b = false;
    boolean c = a && b; // c recebe falso
    System.out.println(c);
    c = a || b; // c recebe verdadeiro
    System.out.println(c);
    c = !a; // c recebe falso
    System.out.println(c);
    c = a ^ b; // c recebe verdadeiro
    System.out.println(c);
  }
}
